### PR TITLE
PLATFORM-3381: Updates for secondary claims

### PR DESCRIPTION
**Pull request by baldwindc**

### CODE REVIEW

File: source/includes/_claims.md [hunk 1]
```diff
@@ -3169,9 +3489,111 @@ The /claims/ response contains an activity and thus returns the same object as t
...
+| 'contractual_obligations' |
+| 'other_adjustments' |
+| 'payor_initiated_reductions' |
+| 'patient_responsibility' |
+| 'corrections_and_reversals' |
+
+
 <a name="claim-frequency"></a>
 Full list of possible values that can be used in the claim.claim_frequency parameter on the claim:
-
```
Comment (wootencl) [inline source/includes/_claims.md]: Did you mean to delete this new line? It'll actually mess with the below markdown table it not there.

File: source/includes/_claims.md [hunk 2]
```diff
@@ -3169,9 +3489,111 @@ The /claims/ response contains an activity and thus returns the same object as t
...
+|:---------------------------------- |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| claim_adjustment_group_code        | Group code describing the type of adjustment [object](#claim-adjustment-group-codes) |
+| claim_adjustment_reason_code       | Required for Secondary: Reason code as provided in the 835 response from the primary payer. |
+| adjustment_amount                  | Required for Secondary: Adjustment amount as specified for the secondary payer. |
+| adjustment_quantity                | Required for Secondary: Adjustment quantity as specified for the secondary payer. |
+
+
+<a name="claim-adjustment-group-codes"></a>
```
Comment (wootencl) [inline source/includes/_claims.md]: Are there suppose to be descriptions here?

**Comment by wootencl** [comment]
One extremely minor nitpick. So far I've been putting a colon after an object (i.e.- `Amazing Object:`). Not a huge deal but might be something to add for consistency.

File: source/includes/_claims.md (hunk 1 above)
Comment (baldwindc) [inline source/includes/_claims.md]: I did not, thank you.

File: source/includes/_claims.md (hunk 2 above)
Comment (baldwindc) [inline source/includes/_claims.md]: No, but I deleted the headers.

**Comment by baldwindc** [comment]
@wootencl Can you give me a specific example from the docs?

**Comment by wootencl** [comment]
@baldwindc Sure. Looking at your demo if you look at one of the object models you added the intro/title to the object, `Coordination of Benefits object:`, <- there's a colon at the end. Compared to the below object, `Other Subscriber object`, <- no colon. Really minor difference.

**Pull request merged**